### ISSUE TITLE
Error 500

### ISSUE THREAD

**Issue by imatpro** [marcan/pacemaker-exporter#1]
Hi, thanks for this exporter.

Error code: 500

Message: 'module' object has no attribute 'run'.

Error code explanation: 500 - Server got itself in trouble.

on exporter page

I habe python3, prometheus python client and crm_mon

#dpkg -l |grep pacemaker
ii  crmsh                          2.2.0-1~bpo8+1                     amd64        CRM shell for the pacemaker cluster manager
ii  pacemaker                      1.1.15-3~bpo8+1                    amd64        cluster resource manager
ii  pacemaker-cli-utils            1.1.15-3~bpo8+1                    amd64        cluster resource manager command line utilities
ii  pacemaker-common               1.1.15-3~bpo8+1                    all          cluster resource manager common files
ii  pacemaker-resource-agents      1.1.15-3~bpo8+1                    all          cluster resource manager general resource agents

prometheus-client (0.1.1)

Thanks for helping

**Comment by marcan** [comment]
Sorry, I didn't specify the dependencies tightly enough. subprocess.run requires Python 3.5 or newer. Presumably you have 3.4.

Try this patch. If it works I'll commit it to improve compatibility.

```diff
diff --git a/prometheus-pacemaker-exporter.py b/prometheus-pacemaker-exporter.py
index c51253a..300c612 100755
--- a/prometheus-pacemaker-exporter.py
+++ b/prometheus-pacemaker-exporter.py
@@ -31,8 +31,7 @@ def p_bool(b):
     return 1 if b in ('true', '1') else 0
 
 def get_xml():
-    return subprocess.run(["crm_mon", "-X"], check=True,
-                          stdout=subprocess.PIPE).stdout
+    return subprocess.check_output(["crm_mon", "-X"])
 
 class PacemakerCollector(object):
     def __init__(self, args):

```

**Comment by imatpro** [comment]
Thanks for quick answer,

Perfect, it's working with python3.4

I have error 500 in localhost:9202 but /metrics page working

but, it's normal metric gauge as format 1.0 or 0.0

ex : 
```
# HELP pacemaker_resource_node Whether a resource is running on each node
# TYPE pacemaker_resource_node gauge
pacemaker_resource_node{id="VIP-Slave2",instance="",node="db-02"} 1.0
pacemaker_resource_node{id="VIP-Slave2",instance="",node="db-01"} 0.0
```

Thanks you very much Marcan

**Comment by marcan** [comment]
Fixed in de6f837 (the root page should work now too).
